torch/testing/_asserts.py

```diff
@@ -246,10 +322,14 @@ def _trace_mismatches(actual: Tensor, expected: Tensor, mismatches: Tensor) -> D
 
 
 @_check_complex_components_individually
+@_check_sparse_coo_members_individually
+@_check_sparse_csr_members_individually
 def _check_values_equal(
     actual: Tensor,
     expected: Tensor,
     *,
+    rtol: float = 0.0,
+    atol: float = 0.0,
```

Comment:
@mruberry This is just a temporary solution to match the signatures of `_check_values_equal` and `_check_values_close`. #58918 will resolve this properly.